openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/Assignees.tsx

```diff
@@ -35,6 +38,26 @@ const Assignees: FC<Props> = ({ assignees, onSearch, onChange, options }) => {
     onChange(newValues as Option[]);
   };
 
+  const tagRender = ({
+    label,
+    closable,
+    onClose,
+    value,
+  }: CustomTagProps): React.ReactElement => {
+    return (
+      <Tag className="assignee-tag" closable={closable} onClose={onClose}>
+        <Space>
+          <ProfilePicture
+            id={value}
+            name={label as unknown as string}
+            width="22"
+          />
+          <span>{label}</span>
+        </Space>
+      </Tag>
```

Comment:
Can we try this here: https://ant.design/components/tag/#components-tag-demo-icon